src/main/java/org/fisco/bcos/sdk/channel/model/EnumNodeVersion.java

```diff
@@ -0,0 +1,108 @@
+package org.fisco.bcos.sdk.channel.model;
+
+public enum EnumNodeVersion {
+    BCOS_2_0_0("2.0.0"),
+    BCOS_2_1_0("2.1.0");
+
+    private String version;
+
+    private EnumNodeVersion(String version) {
+        this.version = version;
+    }
+
+    public String getVersion() {
+        return version;
+    }
+
+    public void setVersion(String version) {
+        this.version = version;
+    }
+
+    // the object of node version
+    public class Version {
+        private int major;
+        private int minor;
+        private int patch;
+        private String ext;
+
+        @Override
+        public String toString() {
+            return "Version [major="
+                    + major
+                    + ", minor="
+                    + minor
+                    + ", patch="
+                    + patch
+                    + ", ext="
+                    + ext
+                    + "]";
+        }
+
+        public int getMajor() {
+            return major;
+        }
+
+        public void setMajor(int major) {
+            this.major = major;
+        }
+
+        public int getMinor() {
+            return minor;
+        }
+
+        public void setMinor(int minor) {
+            this.minor = minor;
+        }
+
+        public int getPatch() {
+            return patch;
+        }
+
+        public void setPatch(int patch) {
+            this.patch = patch;
+        }
+
+        public String getExt() {
+            return ext;
+        }
+
+        public void setExt(String ext) {
+            this.ext = ext;
+        }
+    }
+
+    public static Version getClassVersion(String version) throws ChannelPrococolExceiption {
```

Comment:
Add support for sm node version number format, eg: 2.4.0 gm

Reply:
if (version.endsWith("gm")) {
                version = version.substring(0, version.indexOf("gm"));
            }